site/envoy_versions.json

```diff
@@ -1,11 +1,11 @@
 {
   "latestVersion": "1.18.3",
   "versions": {
-    "1.14.6": {
```

Comment:
I replaced this not added because we've not yet released a version of getenvoy that reads this file. When we do, we should encourage use of latest patches.